test/amazonlinux/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM amazonlinux
```

Comment:
Should I lock this at 2017.12 or leave it at latest?
(I'm inclined to locking)

Reply:
I think latest is fine, as we want to know that new changes in Amazon Linux don't break our code. In the future, we'll prob add support for Amazon Linux 2.

Reply:
* after discussion on slack, leaving at `latest^